euler_support/urdf/euler_macro.xacro

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="euler" params="prefix">
+        <xacro:include filename="$(find motoman_mh5lf_support)/urdf/mh5lf_macro.xacro" />
+        <xacro:motoman_mh5lf prefix="${prefix}motoman_" />
+        <!--xacro:include filename="$(find euler_support)/urdf/asus_xtion_pro.xacro" /-->
+        <xacro:include filename="$(find industrial_extrinsic_cal)/urdf/asus_sensor_cal_macro.xacro" />
+        <xacro:asus_sensor_definitions prefix="${prefix}asus_" parent_link="euler_motoman_base_link" />
```

Comment:
Using a trailing underscore in the prefix ```"${prefix}asus_"``` causes the name of the link to have two underscore characters.  For instance the name of the frame link gets resolved to "euler_asus__frame".
Also, the asus sensor isn't showing in the right location.

Reply:
I just deleted the underscore at the end of "asus"  Let me know if that wasn't what you intended.

Reply:
I just wanted to avoid having 2 underscore characters as in "euler_asus__frame".